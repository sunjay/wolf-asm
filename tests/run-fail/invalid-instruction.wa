# This program has some valid instructions and invalid instructions

section .code

nop
add $1, $2
# Not a valid instruction (runtime exception)
.b8 0xF0F0F0F0F0F0F0F0
# These instructions will not run
nop
sub $1, $1
