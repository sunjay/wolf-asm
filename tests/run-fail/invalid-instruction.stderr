Error: Failed to execute instruction at `0x10`

Caused by:
    Invalid instruction: opcode `3855` is not supported
